app/assets/javascripts/settings.js.erb

```diff
@@ -27,8 +27,18 @@ See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
 
-jQuery(document).ready(function($) {
-  $('#settings_session_ttl_enabled').on('change', function(){
-    $('#settings_session_ttl_container').toggle($(this).is(':checked'));
-  }).trigger('change');
-});
+(function($) {
+  $(function() {
+
+    $('#settings_session_ttl_enabled').on('change', function(){
+      $('#settings_session_ttl_container').toggle($(this).is(':checked'));
+    }).trigger('change');
+
+    $('#settings_repositories_automatic_managed_repos').change(function() {
+      var checked = this.checked;
+      $('#settings_repositories_automatic_managed_vendor').prop('disabled', function() {
```

Comment:
Apparently I can't see a visual difference between an enabled and a disabled select box.
This is quite annoying since I clicked three times on the select box until I realized that I need to check that checkbox before.

How about providing a `disabled` option in the select box and not offering the checkbox at all?

E.g.

> Choose an SCM type for automatic creation
> * do not create repos automatically
> * Git
> * SVN